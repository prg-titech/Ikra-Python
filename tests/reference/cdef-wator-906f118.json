{
    "basename": "cdef-wator-906f118",
    "cmd": "new-src/interface.py --emit-cdef {infile}",
    "infile": "examples/wator/wator.py",
    "infile_hash": "881d065157c07b9b943434cefd615669afe6d54e10eaf935cd8bbc61",
    "stdout": "cdef-wator-906f118.stdout",
    "stdout_hash": "89827d237ee22c898f524a665cb7cca2d8cafd70b11f5815b61c68ed",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}