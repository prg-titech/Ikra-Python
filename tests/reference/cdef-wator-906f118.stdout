pygame 2.0.1 (SDL 2.0.14, Python 3.7.5)
Hello from the pygame community. https://www.pygame.org/contribute.html

int Cell_Cell_setup();
int Cell_Cell_prepare();
int Fish_Fish_prepare();
int Cell_Cell_decide();
int Fish_Fish_update();
int Shark_Shark_prepare();
int Shark_Shark_update();
int parallel_new_Cell(int object_num);
int AllocatorInitialize();
int Fish_do_all(void (*pf)(int));
int Shark_do_all(void (*pf)(int, int));
int Cell_do_all(void (*pf)(int, int, int, int, int, int));
int AllocatorUninitialize();
